### ISSUE TITLE
z-indices change on running build

### ISSUE THREAD

**Issue by JoostKiens** [kaliberjs/build#52]
How to reproduce: in a css module, set a z-index to a high number: 
```
z-index: 100000;
```

`Run build` and see the number change to something lower: probably 1.
This is troublesome when trying to position something with a z-index set through a different mechanism, eg javascript

**Comment by EECOLOR** [comment]
`{zindex: false}`

**Comment by EECOLOR** [comment]
https://github.com/ben-eb/gulp-cssnano/issues/8
https://github.com/ben-eb/cssnano/issues/88
https://github.com/ben-eb/gulp-cssnano/issues/14
http://cssnano.co/optimisations/zindex/

**Comment by EECOLOR** [comment]
ok, so the documentation online is for version 4 which is currently a release candidate: https://github.com/ben-eb/cssnano/issues/408

Looking at the source code, it seems that safe options would fix this issue and fix #51 as well.

```
let safeOptions = {
    postcssConvertValues: {
        length: false,
    },
    postcssDiscardUnused: {
        disable: true,
    },
    postcssMergeIdents: {
        disable: true,
    },
    postcssReduceIdents: {
        counterStyle: false,
        gridTemplate: false,
        keyframes: false,
    },
    postcssNormalizeUrl: {
        stripWWW: false,
    },
    postcssZindex: {
        disable: true,
    },
};
```

When version 4 is released a similar set is the default, there seems to be only exception: https://github.com/ben-eb/cssnano/issues/409
